### ISSUE TITLE
Question on sending scripted events to WAM

### ISSUE THREAD

**Issue by rhelsing** [webaudiomodules/wam-examples#86]
I’m trying to set up a web based sequencer essentially and I’m wondering how feasible it would be to send note on/note off midi type events from the sequencer to a wasm based synth? Is there a javascript accessible event api to send these signals?

**Comment by owengc** [comment]
Hello and thanks for your interest! 

Yes, the WAM API has been designed to facilitate exactly this sort of interaction. I invite you to take a look at the WAM [API](https://github.com/webaudiomodules/api) and [SDK](https://github.com/webaudiomodules/sdk) repos, as well as the simple [host example](https://github.com/webaudiomodules/wam-examples/blob/master/packages/host/main.js#L504-L505) found here in wam-examples to get a sense of how these things work.

In short, if your synth is a WAM you can use your sequencer to schedule midi events from either the main thread (asynchronous, requires scheduling in advance) or directly on the audio thread (synchronous, can be scheduled 'just in time', but requires that your sequencer is itself also a WAM / AudioWorklet node). Please let us know if you have further questions.
